docs/modules/reference/pages/configuration/provisioning/detectors/HttpsDetector.adoc

```diff
@@ -14,5 +14,5 @@ Use this detector to find and assign services based on HTTPS.
 
 .Parameters for the HTTPS service detector
 |===
-| Parameters for the HTTPS detector are the same parameters as the <<provisioning/detectors/HttpDetector.adoc#HttpDetector, HttpDetector>>.
+| Parameters for the HTTPS detector are the same parameters as the <<reference:configuration/provisioning/detectors/HttpDetector.adoc#HttpDetector, HttpDetector>>.
```

Comment:
This line does not need to change, as it is a relative link within the same Reference section.
```suggestion
| Parameters for the HTTPS detector are the same parameters as the <<provisioning/detectors/HttpDetector.adoc#HttpDetector, HttpDetector>>.
```